platform-api/spec/impls/gateway-management/artifacts/openapi-gateways.yaml

```diff
@@ -77,6 +78,47 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /gateways/status:
```

Comment:
/gateways/status and   /gateways/{id} are both GET calls which conflicts each other. Generally its not recommended to have such resources as it can cause incosistent behaviours. (when fronted via gateways/LBs with resource policies etc they can conflict)

Shall we incoporate this into /gateways itself? (include the status inside each gateway model)

Reply:
Cool